tests/integration/test_put_along_axis.py

```diff
@@ -54,6 +67,191 @@ def test_ndim(ndim):
         assert np.array_equal(np_a, num_a)
 
 
+def test_full():
+    shape = (3, 4, 5)
+    ndim = len(shape)
+    np_arr = mk_seq_array(np, shape)
+    num_arr = mk_seq_array(num, shape)
+
+    for axis in range(-1, ndim):
+        size = shape[axis]
+        axis_values = (0, size - 1, size * 2)
+
+        for shape_idx in broadcasts_to_along_axis(shape, axis, axis_values):
+            np_indices = mk_seq_array(np, shape_idx) % shape[axis]
+            num_indices = mk_seq_array(num, shape_idx) % shape[axis]
+            np_a = np_arr.copy()
+            num_a = num_arr.copy()
+            np.put_along_axis(np_a, np_indices, 100, axis=axis)
+            num.put_along_axis(num_a, num_indices, 100, axis=axis)
+            assert np.array_equal(np_a, num_a)
+
+
+def test_values():
+    shape = (3, 4, 5)
+    np_arr = mk_seq_array(np, shape)
+    num_arr = mk_seq_array(num, shape)
+    shape_idx = (3, 4, 5)
+    axis = 0
+    np_indices = mk_seq_array(np, shape_idx) % shape[axis]
+    num_indices = mk_seq_array(num, shape_idx) % shape[axis]
+
+    for shape_values in broadcasts_to(shape_idx):
+        for s in equivalent_shapes_gen(shape_values):
+            np_values = mk_seq_array(np, s)
+            num_values = mk_seq_array(num, s)
+            np_a = np_arr.copy()
+            num_a = num_arr.copy()
+            np.put_along_axis(np_a, np_indices, np_values, axis=axis)
+            num.put_along_axis(num_a, num_indices, num_values, axis=axis)
+            assert np.array_equal(np_a, num_a)
+
+
+def test_empty_indice():
```

Comment:
did you miss `s` in `indices`?

Reply:
No. In test_values, `indices` has fixed shape (3, 4, 5) while the shape of `values` varies.